[RFE] Add 'data' property to ValidationStream interface

Messages emitted on `validationReceived` stream have `data` property, but `xrpl.ValidationStream` interface does not include this property. 
```
{
  cookie: '10019730095899181630',
  data: '228000000126046F0B01292AB47C7A3A8B0D3B6F775C823E5148E2C637974E505E81356D178EBE80834B633AC426B29CAB5DB42A9C073FF99B50177B68F88701CD73D951976540DFF5CAA2A00F6B605F1BCE82DC1B5D83DE1CBCDF5019E56337278151F76351C8251392FD2089C9714B92A3BC38A0CF720D7B02F69E517321033CE387D979DAEAF4A7D67F0A76C84B2A09C3859558A1DAE886663D3F0121A9D77646304402202A20BDBFA3C2BC376A46ED7A3A245F8D19D1DABE96888D84429B6F732525E46D022027F9DAA466A63E44938BC074289804EB5B68BC3F1F99988E384213526476143C',
  flags: 2147483649,
  full: true,
  ledger_hash: '48E2C637974E505E81356D178EBE80834B633AC426B29CAB5DB42A9C073FF99B',
  ledger_index: '74386177',
  master_key: 'nHB6KGx8YNddfLwTKhLyqytFLyhjmh9vvv8LpUfBg53S8BZbYyQn',
  signature: '304402202A20BDBFA3C2BC376A46ED7A3A245F8D19D1DABE96888D84429B6F732525E46D022027F9DAA466A63E44938BC074289804EB5B68BC3F1F99988E384213526476143C',
  signing_time: 716471418,
  type: 'validationReceived',
  validated_hash: 'E56337278151F76351C8251392FD2089C9714B92A3BC38A0CF720D7B02F69E51',
  validation_public_key: 'n9Lxh4VJhbpThQqpXBwC1rGL4t93zqVeEetbJKSyAEBuhdfBbFmc'
}
```

This causes Typescript to print `error TS2339: Property 'data' does not exist on type 'ValidationStream'.` errors.

It would be great to have `data` in ValidationStream interface - https://github.com/XRPLF/xrpl.js/blob/main/packages/xrpl/src/models/methods/subscribe.ts#L183

--- a/packages/xrpl/HISTORY.md
+++ b/packages/xrpl/HISTORY.md
@@ -4,6 +4,7 @@ Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xr
 ## Unreleased
 ### Added
 * Support for ExpandedSignerList amendment that expands the maximum signer list to 32 entries.
+* Add `cookie` and `data` to `ValidationStream` interface
 
 ## 2.4.0 (2022-09-01)
 ### Added

--- a/packages/xrpl/src/models/methods/subscribe.ts
+++ b/packages/xrpl/src/models/methods/subscribe.ts
@@ -190,6 +190,18 @@ export interface ValidationStream extends BaseStream {
   /** The amendments this server wants to be added to the protocol. */
   base_fee?: number
   /**
+   * An arbitrary value chosen by the server at startup.
+   *
+   * If the same validation key pair signs validations with different cookies
+   * concurrently, that usually indicates that multiple servers are incorrectly
+   * configured to use the same validation key pair.
+   */
+  cookie?: string
+  /**
+   * The contents of the validation message in its canonical binary form
+   */
+  data?: string
+  /**
    * The unscaled transaction cost (reference_fee value) this server wants to
    * set by Fee voting.
    */

